requirements.txt

```
@@ -1,3 +1,7 @@
+# Version updates managed by pyup.io
+
+dynaconf==3.1.2
```

Use latest available version for dynaconf
```suggestion
dynaconf==3.1.4
```